php-frontend/src/main/java/org/sonar/php/parser/LexicalConstant.java

```diff
@@ -82,27 +82,27 @@
   private static final String NON_SPECIAL_CHARACTERS = "(?:[^\"\\\\$\\{])";
   private static final String NON_SPECIAL_CHARACTERS_EXECUTION = "(?:[^\\\\$\\{`])";
   private static final String NON_SPECIAL_CHARACTERS_HEREDOC = "(?:[^\\\\$\\{])";
-  private static final String ESCAPED_CHARACTERS = "(?:\\\\[\\s\\S])";
+  private static final String ESCAPED_CHARACTERS_AND_STANDALONE_BACKSLASH = "(?:\\\\[\\s\\S]?)";
```

Comment:
I would rename it to `ESCAPED_CHARACTER_OR_STANDALONE_BACKSLASH`: I think it would be clearer.